disnake/interactions/base.py

```diff
@@ -1211,18 +1234,63 @@ class InteractionMessage(Message):
     __slots__ = ()
     _state: _InteractionMessageState
 
+    @overload
     async def edit(
         self,
         content: Optional[str] = MISSING,
```

Comment:
overloads should be using `...` for default parameter values

Reply:
https://github.com/DisnakeDev/disnake/pull/469/commits/3b75236afe85921e2d2a50fef002f3b412313e30